cmd/kms-rest/startcmd/start.go

```diff
@@ -263,8 +263,7 @@ func prepareKMSCreator(kmsStorageProvider ariesstorage.Provider) operation.KMSCr
 }
 
 func prepareSecretLock(passphrase string) (secretlock.Service, error) {
-	salt := randomBytes(keySize)
-	return hkdf.NewMasterLock(passphrase, sha256.New, salt)
+	return hkdf.NewMasterLock(passphrase, sha256.New, nil)
```

Comment:
Noted a TODO in aries-framework-go that we should consider a better name for "masterlock" and "master key".

Reply:
I vote for `primaryKey`, and as the second option - `mainKey` as was suggested by @Baha-sk some time ago